ner_v2/detectors/numeral/constant.py

```diff
@@ -10,7 +10,6 @@
 NUMBER_TYPE_UNIT = 'unit'
 NUMBER_TYPE_SCALE = 'scale'
 
-
```

Comment:
For my understanding, what kind of data are we supposed to store in these two folders(ner_v1 and ner_v2)?

Reply:
ner v1 was older version.
Some entity detection logics like regex and email were retained while newer logics were brought up in ner v2. 
We can see which is called where by checking API calls on haptik API side